roles/update-pkgs/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+- name: Update packages and kernel to latest available versions
+  package:
+    name: '*'
+    state: latest
+  when: ansible_pkg_mgr in ['yum', 'dnf']
+
+- name: Check if reboot required
+  shell: needs-restarting -r
+  register: reboot_check
+  ignore_errors: yes
+  when: ansible_distribution in ['CentOS', 'RHEL']
+
+- name: Reboot if necessary
+  reboot:
+    reboot_timeout: 600
```

Comment:
I see default value is already `600`, so wondering if you want to mention the default value here?

Reply:
Did away with the defaults..